app/scripts/legacy/phase1_multimodal_pipeline.py

```
@@ -0,0 +1,2322 @@
+                    )
+                    chunks.append({"id": cid, "content": text, "metadata": meta})
+                    stats["table_chunks"] += 1
+
+            # Embedded images in Excel sheets
+            images = list(getattr(ws, "_images", []))
+            image_block_start = max(1, math.ceil(len(data_lines) / max(1, args.table_rows_per_chunk)))
+            for img_i, img in enumerate(images, 1):
+                try:
+                    image_bytes = img._data()
+                except Exception:
+                    continue
```

Ruff (0.15.5)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)